### PR TITLE
add auto install in case of missing contrib

### PR DESCRIPTION
**Pull request by hkollmann**
I case of missing contrib call qx contrib install to install the contrib

### CODE REVIEW

File: lib/qx/tool/cli/commands/Compile.js [hunk 1]
```diff
@@ -472,6 +472,10 @@ qx.Class.define("qx.tool.cli.commands.Compile", {
           });
         }
       });
+
+      if (!data.libraries.every(libData => fs.existsSync(libData + "/Manifest.json"))) {
+        await (new qx.tool.cli.commands.contrib.Install({quiet:true})).process();
```
Comment (cboulanger) [inline lib/qx/tool/cli/commands/Compile.js]: Maybe inform the user about implicit installation unless a "quiet" flag has been set?

Comment (hkollmann) [reply lib/qx/tool/cli/commands/Compile.js]: we have no message in auto update and list either. Should'nt we inform the user in this case too?

Comment (hkollmann) [reply lib/qx/tool/cli/commands/Compile.js]: Done

**Comment by cboulanger** [comment]
2 approvals (fast-track procedure), merging. Thank you @hkollmann !!

**Pull request merged**